config/locales/en.yml

```diff
@@ -1117,6 +1117,11 @@ en:
         question: Confirm that your employee works at the location below
         errors:
           presence: Select the nursery where your employee works
+      paye_reference:
+        question: "What is %{nursery_name}'s employer PAYE reference?"
```

Comment:
from my experience there is typically a preference to use smart quotes where possible including apostrophes https://designnotes.blog.gov.uk/2015/09/16/tips-for-creating-good-typography/

Reply:
ah, interesting - we have both in the codebase, but mostly the smart quote.